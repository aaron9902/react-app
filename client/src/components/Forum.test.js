import React from 'react';
import { mount } from 'enzyme';
import Forums from './forums';
import { BrowserRouter, BrowserRouter as Router, Link } from "react-router-dom"; 

describe('<Forums/ >', () => {
  
  it('matches snapshot', () => {
    const wrapper = mount(<BrowserRouter><Forums /></BrowserRouter>);
    expect(wrapper).toMatchSnapshot();
  });

  it('it has input', () => {
    const wrapper = mount(<BrowserRouter><Forums /></BrowserRouter>);
    let input = wrapper.findWhere(
      node => node.type() === 'input'
    );
    expect(input.exists()).toBe(true);
  });

});
