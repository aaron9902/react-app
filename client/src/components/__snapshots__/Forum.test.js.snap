// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Forums/ > matches snapshot 1`] = `ReactWrapper {}`;
